roles/hosted_engine_setup/tasks/create_storage_domain.yml

```diff
@@ -123,21 +123,16 @@
       patterns: ^.*.(?<!meta).ovf$
       use_regex: true
     register: app_ovf
-  - name: Parse OVF
-    xml:
-      path: "{{ app_ovf.files[0].path }}"
-      xpath: /ovf:Envelope/Section/Disk
-      namespaces:
-        ovf: http://schemas.dmtf.org/ovf/envelope/1/
-        rasd: http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData
-        vssd: http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData
-        xsi: http://www.w3.org/2001/XMLSchema-instance
-      content: attribute
-    register: disk_size_xml
+  - name: Get ovf data
+    command: cat "{{ app_ovf.files[0].path }}"
+    register: ovf_data
```

Comment:
Please replace the command with file lookup. https://docs.ansible.com/ansible/latest/plugins/lookup.html